lib/YaST/Bootloader/BootloaderOptionsTab.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces all accessing methods for Bootloader Options tab
```

Comment:
"bootloader options tab" is jut the one tab more on the right, it is BootloaderSettingsTabs the object to map.

Reply:
it should have the name suggested above.
you are missing to create some `new()` method and you can drop the `init()`